builder/commands_test.go

```diff
@@ -0,0 +1,27 @@
+package builder
+
+import (
+	"testing"
+	"time"
+
+	"github.com/Masterminds/cookoo"
+)
+
+func TestSleep(t *testing.T) {
+	reg, router, cxt := cookoo.Cookoo()
+
+	reg.Route("test", "Test route").
+		Does(Sleep, "res").Using("duration").WithDefault(3 * time.Second)
+
+	start := time.Now()
+	err := router.HandleRequest("test", cxt, true)
+	if err != nil {
```

Comment:
This `err != nil` could be collapsed with the previous line, which we seem to prefer whenever possible. (So there's one style thing that isn't enforced by our linting tools--ping @kalbasit --but therefore also isn't strictly required.)